php/php.editor/src/org/netbeans/modules/php/editor/parser/astnodes/CastExpression.java

```diff
@@ -27,7 +27,7 @@ public class CastExpression extends Expression {
 
     public enum Type {
         INT, // 'int'
-        REAL, // 'real'
+        FLOAT, // 'float'
```

Comment:
I don't think we should change this because this is an API.

@tmysik What do you think?

Reply:
Yes, see [my comment](https://github.com/apache/netbeans/pull/7387#discussion_r1608051235).